tipgstac/models.py

```diff
@@ -78,6 +78,63 @@ def validate_bbox(cls, v: BBox):
         return v
 
 
+class CollectionsSearch(BaseModel):
+    """PgSTAC Collections Search Query model."""
+
+    ids: Optional[List[str]] = None
+    bbox: Optional[BBox] = None
+    intersects: Optional[Geometry] = None
+    query: Optional[Dict[str, Dict[Operator, Any]]] = None
+    filter: Optional[Dict] = None
+    datetime: Optional[str] = None
+    sortby: Optional[Any] = None
+    fields: Optional[Dict[str, Set]] = None
+    filter_lang: Annotated[Optional[FilterLang], Field(alias="filter-lang")] = None
+    limit: Optional[int] = None
+    offset: Optional[int] = None
+    conf: Optional[Dict] = None
```

Comment:
@bitner it's not clear to me what is supported by pgstac collection search model